libs/platform/router/src/services/default-router.service.ts

```diff
@@ -48,6 +48,7 @@ export class DefaultRouterService implements RouterService {
     this.router$.next(url[0]);
     this.urlSearchParams$.next(queryParams);
     this.routerEvents$.next({ route, type: RouterEventType.NavigationEnd });
+    globalThis.scrollTo?.(0, 0);
```

Comment:
can we make this work with https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior? It means that JS code only scrolls to the position, and css wil dicate the behviaor